apps/client/components/Recent/History/History.hooks.ts

```diff
@@ -0,0 +1,81 @@
+import { useQueryClient } from "@tanstack/react-query";
+import { useRouter } from "next/router";
+import { diets } from "~/constants/data/common";
+import {
+  useCurrentSelectCategorySetter,
+  useCurrentSelectKeywordSetter,
+  useMapBoundsValue,
+  useMapSearchByCountrySetter
+} from "~/recoil/atoms";
+import {
+  Keys,
+  useDeleteRecentHistory,
+  useFetchEatingHabitByFiltering,
+  useFetchRecentHistory,
+  useFetchRestaurantByCountry
+} from "~/server/recent";
+
+const useHistory = () => {
+  const { push } = useRouter();
+  const queryClient = useQueryClient();
+  const setCurrentSelectKeyword = useCurrentSelectKeywordSetter();
+  const setCurrentSelectCategory = useCurrentSelectCategorySetter();
+  const { data: recentHistories } = useFetchRecentHistory();
```

Comment:
history들을 불러오는것이니 
```suggestion
  const { data: recentHistories } = useFetchRecentHistories();
```

라고 명명하는것은 어떠신가요?!

Reply:
[ix: history들을 fetch해올 때 여러 history들을 가져오므로 histories로 변경](https://github.com/toppingskorea/toppings-client/pull/43/commits/92c6ba13e009d8f481a4461e0ebd80015cd6cbc7)

감사합니다. 수정완료하였습니다!